lusid-sdk-python/tests/test_transactions.py

```diff
@@ -0,0 +1,233 @@
+        loader_response = cls.inst_loader.load_instruments(cls)
+
+        assert len(loader_response.values) == 5
+
+    def test_load_listed_instrument_transaction(self):
+
+        uuid_gen = uuid.uuid4()
+        scope = "finbourne"
+        effective_date = datetime(2018, 1, 1, tzinfo=pytz.utc)
+
+        # portfolio request
+        original_portfolio_id = "Id-" + str(uuid_gen)
+        request = models.CreateTransactionPortfolioRequest("Portfolio-" + str(uuid_gen),
+                                                           original_portfolio_id,
+                                                           base_currency="GBP",
+                                                           created=effective_date)
+        # create the portfolio
+        portfolio_response = self.client.create_portfolio(scope, request)
+
+        self.assertEqual(portfolio_response.id.code, request.code)
+
+        portfolio_id = portfolio_response.id.code
+
+        # create the transaction
+        transaction_request = models.TransactionRequest(
+                                    transaction_id=str(uuid.uuid4()),
+                                    type="Buy",
+                                    instrument_uid=
+                                    self.inst_loader.instrument_response.values['correlationId1'].lusid_instrument_id,
+                                    transaction_date=effective_date,
+                                    settlement_date=effective_date,
+                                    units=100.0,
+                                    transaction_price=models.TransactionPrice(12.3),
+                                    total_consideration=models.CurrencyAndAmount(100 * 12.3, "GBP"),
+                                    source="Client")
+        # add the trade
+        upsert_response = self.client.upsert_transactions(scope, portfolio_id, [transaction_request])
+
+        # get the trade again
+        returned_transaction = self.client.get_transactions(scope, portfolio_id )
+        assert returned_transaction.count == 1
+        assert transaction_request.transaction_id == returned_transaction.values[0].transaction_id
+
+    def test_zzload_cash_transactions(self):
+
+        uuid_gen = uuid.uuid4()
+        scope = "finbourne"
+        effective_date = datetime(2018, 1, 1, tzinfo=pytz.utc)
+
+        # portfolio request
+        original_portfolio_id = "Id-" + str(uuid_gen)
+        request = models.CreateTransactionPortfolioRequest("Portfolio-" + str(uuid_gen),
+                                                           original_portfolio_id,
+                                                           base_currency="GBP",
+                                                           created=effective_date)
+        # create the portfolio
+        portfolio_response = self.client.create_portfolio(scope, request)
+
+        self.assertEqual(portfolio_response.id.code, request.code)
+
+        portfolio_id = portfolio_response.id.code
```

Comment:
Same as above, we already have the id in 'original_portfolio_id'

Reply:
removed